packages/chrome/src/elements/sheet/Sheet.tsx

```diff
@@ -0,0 +1,152 @@
+  isAnimated?: boolean;
+  /** Focuses on the Sheet when isOpen is true and mounted **/
+  focusOnMount?: boolean;
+  /** Restores focus to the previous element after the Sheet is closed **/
+  restoreFocus?: boolean;
+  /** Determines the position of the Sheet **/
+  placement?: 'start' | 'end';
+  /** Sets the width in pixels, based on the placement of the Sheet */
+  size?: number;
+}
+
+/**
+ * @extends HTMLAttributes<HTMLElement>
+ */
+// eslint-disable-next-line react/display-name
+export const Sheet = React.forwardRef<HTMLElement, ISheetProps>(
+  ({ id, isOpen, isAnimated, focusOnMount, restoreFocus, size, children, ...props }, ref) => {
+    const sheetRef = useRef<HTMLElement>(null);
+
+    const seed = useUIDSeed();
+    const [idPrefix] = useState<string>(id || seed(`sheet_${PACKAGE_VERSION}`));
+    const titleId = `${idPrefix}--title`;
+    const descriptionId = `${idPrefix}--description`;
+
+    const sheetContext = useMemo(() => ({ idPrefix }), [idPrefix]);
+
+    useFocusableMount({ targetRef: sheetRef, isMounted: isOpen, focusOnMount, restoreFocus });
+
+    if (isAnimated) {
+      return (
+        <SheetContext.Provider value={sheetContext}>
+          <StyledSheet
+            isOpen={isOpen}
+            size={size}
+            isAnimated
+            tabIndex={-1}
+            id={idPrefix}
+            aria-labelledby={titleId}
+            aria-describedby={descriptionId}
+            ref={mergeRefs([sheetRef, ref])}
+            {...props}
+          >
+            <CSSTransition
+              in={isOpen}
+              unmountOnExit
+              timeout={500}
+              classNames="side-sheet-transition"
+            >
+              <StyledSheetWrapper size={size}>{children}</StyledSheetWrapper>
+            </CSSTransition>
+          </StyledSheet>
+        </SheetContext.Provider>
+      );
+    }
+
+    return (
+      <SheetContext.Provider value={sheetContext}>
+        <StyledSheet
+          isOpen={isOpen}
+          size={size}
+          isAnimated={false}
+          tabIndex={-1}
+          id={idPrefix}
+          aria-labelledby={titleId}
+          aria-describedby={descriptionId}
+          ref={mergeRefs([sheetRef, ref])}
+          {...props}
+        >
+          {isOpen ? <StyledSheetWrapper size={size}>{children}</StyledSheetWrapper> : null}
+        </StyledSheet>
+      </SheetContext.Provider>
+    );
+  }
+) as IStaticSheetExport<HTMLElement, ISheetProps>;
+
+Sheet.Title = SheetTitle;
+Sheet.Description = SheetDescription;
+Sheet.Header = SheetHeader;
+Sheet.Body = SheetBody;
+Sheet.Footer = SheetFooter;
+Sheet.FooterItem = SheetFooterItem;
+Sheet.Close = SheetCloseButton;
+
+Sheet.displayName = 'Sheet';
+
+Sheet.propTypes = {
+  id: PropTypes.string,
+  isOpen: PropTypes.bool,
+  isAnimated: PropTypes.bool,
+  focusOnMount: PropTypes.bool,
+  restoreFocus: PropTypes.bool,
+  placement: PropTypes.oneOf(['start', 'end']),
+  size: PropTypes.number
+};
+
+Sheet.defaultProps = {
```

Comment:
`size` should default to `380px` here and all other constants and conditional logic removed from styled components.